autograder/question.py

```diff
@@ -98,17 +102,24 @@ def _score_helper(self, submission, additional_data = {}):
         except AutograderFailError:
             # The question has been failed, no additional output is required.
             pass
+        except AutograderHardFailError:
+            # The question has been failed hard, signal to stop grading.
+            self.stop_grading = True
+            pass
```

Comment:
You don't need a pass if you are doing something.

(Careful, this indicates that you don't know what `pass` does.)